ui/main.qml

```diff
@@ -464,12 +480,17 @@ StatusWindow {
                 onAppLoaded: {
                     applicationWindow.appIsReady = true
                     applicationWindow.storeAppState()
-                    moveToAppMain()
+
+                    // only load appMain if biometrics flow is complete
+                    if(!applicationWindow.biometricFlowPending) {
+                        moveToAppMain()
+                    }
                 }
                 onAccountLoginError: function (error, wrongPassword) {
                     onboardingLayout.unwindToLoginScreen() // error handled internally
                 }
                 onSaveBiometricsRequested: (account, credential) => {
+                    applicationWindow.biometricFlowPending = true
```

Comment:
Where is it reset? I know we can't currently logout and create a new account but hey :)